Description: set undefined PATH_MAX and friends
 Define optional posix limits macros to allow
 building on GNU/Hurd.
Author: Milan Kupcevic <milan@debian.org>
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/libavrdude.h
+++ b/libavrdude.h
@@ -28,6 +28,18 @@
 #include <limits.h>
 #include <stdbool.h>
 
+#ifndef PATH_MAX
+#define PATH_MAX 1024
+#endif
+
+#ifndef MAX_PATH
+#define MAX_PATH 1024
+#endif
+
+#ifndef MAXPATHLEN
+#define MAXPATHLEN 1024
+#endif
+
 /* lets try to select at least 32 bits */
 #ifdef HAVE_STDINT_H
 #include <stdint.h>
